cnames_active.js

```diff
@@ -2801,6 +2801,7 @@ var cnames_active = {
   "styli": "cname.vercel-dns.com", // noCF
   "stylis": "thysultan.github.io/stylis.js",
   "subeditor": "shirtiny.github.io/SubEditor",
+  "submodule": "submodule-js.github.io/site/",
```

Comment:
As the CI error notes, you don't need a trailing slash here
```suggestion
  "submodule": "submodule-js.github.io/site",
```